blueprints/controllers/ikea_e1766/ikea_e1766.yaml

```
@@ -211,6 +252,8 @@ action:
       trigger_action: >-
         {%- if integration_id == "zigbee2mqtt" -%}
         {{ trigger.event.data.new_state.state }}
+        {%- elif integration_id == "zigbee2mqtt_v2" -%}
+        {{ trigger.event.data.new_state.state }}
```

This should be
```suggestion
        {{ trigger.payload }}
```
tested it on my box

---

> This should be
> 
> tested it on my box

Oops. Will fix it. Good Catch